poetassistant/urls.py

```diff
@@ -36,13 +36,15 @@
 from poetassistant.apps.definitions.views import DefinitionSet
 from poetassistant.apps.rhymes.views import RhymeSet
 from poetassistant.apps.thesaurus.views import ThesaurusEntrySet
+from poetassistant.apps.wotd.views import WotdSet
 
 schema_view = get_schema_view(title="Poet Assistant API")
 
 router = routers.DefaultRouter()
 router.register(r'rhymes', RhymeSet, "rhymes")
 router.register(r'thesaurus', ThesaurusEntrySet, "thesaurus")
 router.register(r'definitions', DefinitionSet, "definitions")
+router.register(r'wotd', WotdSet, "wotd")
```

Comment:
See if we can include the pagination info here, or somewhere else.
We can't use the standard pagination, because we have infinite pages.

Reply:
Should be an api like this:
```
/wotd?before=<yyyy-MM-dd>&size=<page size>
```